### DISCUSSION TITLE
MPICH4.1.1 static compile issue: no libmpich.a file generated

### DISCUSSION THREAD

**Discussion by VsniperKS** [pmodels/mpich#6533] [General]
I'm trying to build a HPC cluster with MPICH and Ubuntu 20.04 OS. Cause there is a bug at the default MPICH version (3.3.2) on Ubuntu 20.04([https://jira.lsstcorp.org/browse/DM-24746](https://jira.lsstcorp.org/browse/DM-24746)), i have to upgrade it. And as the NONMEM software I am using requires the static library file libmpich.a, I need to generate this file during compilation. 
But when i using GNU to compile MPICH4.1.1 on Ubuntu 20.04, with flag --enable-static but no libmpich.a file generated，here is the configure command：
_./configure --prefix=/opt/mpich-4.1.1 CFLAGS="-m64" FFLAGS="-m64" --enable-f90 --enable-timer-type=gettimeofday --enable-static=yes 2>&1 | tee c.txt_
Is there something wrong with my command? Or does anyone know how to obtain the libmpich.a file? Thank you very much!

**Comment by raffenet** [comment]
By default MPICH only installs a `libmpi.a` static library file. If you need `libmpich.a` you could perhaps make a symlink in the installation directory.

**Reply by VsniperKS** [reply]
You are right raffenet, after cp `libmpi.a` into `libmpich.a`, i can run my NONMEM job successfully, thank you!

**Comment by puneet336** [comment]
this helps! Thank you .
